### ISSUE TITLE
feat(training): no per-epoch accuracy metric — TrainingLoop records loss only, EvaluateModel records a single point

### ISSUE THREAD

**Issue by treeleaves30760** [treeleaves30760/CodefyUI#202]
Found while assembling the acceptance evidence for #138. That issue asks for "loss/accuracy curves" as the deliverable, and the accuracy half cannot be produced.

## What exists today

`TrainingLoop` records exactly six series (`training_loop_node.py`, the `record_metric` call sites):

| series | step | notes |
|---|---|---|
| `train_loss` | per epoch | |
| `val_loss` | per epoch | only when `val_dataloader` is wired |
| `lr` | per epoch | |
| `patience_counter` | per epoch | only when `early_stopping_patience > 0` |
| `best_epoch` | per epoch | only when `early_stopping_patience > 0` |
| `train_loss_batch` | per batch | only when `batch_metrics` is on |

`EvaluateModel` records one series, `eval_accuracy`, and it is a single point:

```python
# app/nodes/training/evaluate_model_node.py:145
context.log_metric("eval_accuracy", accuracy, 1)
```

So a graph can report *an* accuracy — one number, at the end — and there is no way to plot accuracy against epoch. Loss is the only thing with a curve.

## Why this matters beyond the chart

Accuracy against epoch is the plot people actually read for a classifier. It is where you see the schedule land, where you see overfitting begin, and it is the standard figure in every paper this tool is meant to help reproduce. Validation *loss* is a proxy for it and not a substitute — the two routinely diverge late in training, which is precisely the region a cosine schedule spends most of its time in.

It also makes early stopping weaker than it looks: `early_stopping_patience` monitors validation loss, so a run whose accuracy is still climbing while its loss has started rising (normal with label smoothing off and a long schedule) is stopped at the wrong point.

## Suggested direction

The cheapest version that closes the gap: have `TrainingLoop` compute accuracy over `val_dataloader` alongside the val loss it already computes, and record it as `val_accuracy` per epoch. The forward pass is already happening — this is an `argmax` and a counter, no extra compute — and it only needs to fire when the targets look like class indices, so it can stay silent for regression losses.

Worth considering at the same time:

- Let `EvaluateModel` take a step so several evaluations in one graph do not all write step 1 on top of each other.
- A `monitor` parameter on early stopping (`val_loss` / `val_accuracy`), once an accuracy series exists.

## Impact on #138

The evidence bundle there ships loss and learning-rate curves plus a single final test-accuracy figure, because that is all the node set can emit. Noting it here so the gap is recorded rather than quietly absorbed into how the chart happens to look.
